### DISCUSSION TITLE
Creating https links. SSL

### DISCUSSION THREAD

**Discussion by RRozi** [flet-dev/flet#3210] [Q&A]
### Question

When using WEB FLET in Telegram WebApps on android, the error net::ERR_CLEARTEXT_NOT_PERMITTED occurs and the application does not start, as well as in the Web version of telegram WebApps does not start

upd 1. You need to run on the https protocol
upd 2. Also, when clicking on the link to the application in the console of my server, the following WARNING is spammed: Invalid HTTP request received.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by Mo4uToc** [comment]
check this 
https://blog.peddals.com/en/flet-webapp-behind-apache-by-reverse-proxy/
